### ISSUE TITLE
Add ARIA to make it accessible

### ISSUE THREAD

**Issue by nattarnoff** [jaredreich/notie#52]
Currently the alerts don't get picked up by screen readers like VoiceOver. Adding in ARIA will enable this to be a tool used by anyone with assistive technology, not just visual users.

Each dialog with a button in it should have a `role="alertdialog"`,if it is just informational it should get `role="dialog"`. Each dialog needs `aria-labelledby="ID OF TITLE"` and `aria-describedby="ID OF CONTENT"`

For `role="dialog"` the focus must move to the title of the dialog allowing it to be read, then returned to the trigger once dismissed. 

For `role="alertdialog"`, there must be one focusable element in the dialog (button or link perhaps) and focus goes to the title of the dialog. Upon dismissal if the action on the elements inside the dialog do not direct the user to a new page, they should return the user to the trigger element.

Reference: 
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alertdialog_role
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_dialog_role

**Comment by jaredreich** [comment]
Good idea, will implement. Thank you.

**Comment by jaredreich** [comment]
See #55
